### PR TITLE
Fix terminology

### PR DESCRIPTION
**Pull request by sstrigler**
Replace all occurrences or mentions of 'Jabber' with 'XMPP'

### CODE REVIEW

File: xep-0060.xml [hunk 1]
```diff
@@ -243,7 +243,7 @@
     <version>1.0</version>
     <date>2003-10-28</date>
     <initials>psa</initials>
-    <remark><p>Per a vote of the Jabber Council, advanced status to Draft.</p></remark>
+    <remark><p>Per a vote of the XMPP Council, advanced status to Draft.</p></remark>
```
Comment (SamWhited) [inline xep-0060.xml]: I wouldn't change the release notes; at the time of writing it was still the "Jabber Council" (now it's the XSF Council).

File: xep-0060.xml [hunk 2]
```diff
@@ -1141,7 +1141,7 @@ And by opposing end them?
...
     <section3 topic='Request' anchor='subscriber-subscribe-request'>
-      <p>When a Jabber entity wishes to subscribe to a node, it sends a subscription request to the pubsub service. The subscription request is an IQ-set where the &lt;pubsub/&gt; element contains one and only one &lt;subscribe/&gt; element. The &lt;subscribe/&gt; element SHOULD possess a 'node' attribute specifying the node to which the entity wishes to subscribe. The &lt;subscribe/&gt; element MUST also possess a 'jid' attribute specifying the exact XMPP address to be used as the subscribed JID -- often a bare JID &BAREJID; or full JID &FULLJID;.</p>
+      <p>When a XMPP entity wishes to subscribe to a node, it sends a subscription request to the pubsub service. The subscription request is an IQ-set where the &lt;pubsub/&gt; element contains one and only one &lt;subscribe/&gt; element. The &lt;subscribe/&gt; element SHOULD possess a 'node' attribute specifying the node to which the entity wishes to subscribe. The &lt;subscribe/&gt; element MUST also possess a 'jid' attribute specifying the exact XMPP address to be used as the subscribed JID -- often a bare JID &BAREJID; or full JID &FULLJID;.</p>
```
Comment (SamWhited) [inline xep-0060.xml]: Should read: "an XMPP entity"

File: xep-0060.xml [hunk 3]
```diff
@@ -481,7 +481,7 @@ And by opposing end them?
   </ul>
 
         <!-- =================== MAYS and SHOULDS =================== -->
-  <p>Some of the possible uses of a Jabber-based pubsub service will require other features, but these features are OPTIONAL and therefore not mandatory for compliance with this specification. However, if these features are implemented, they MUST adhere to the protocol described herein in to be compliant. These features include:</p>
+  <p>Some of the possible uses of a XMPP-based pubsub service will require other features, but these features are OPTIONAL and therefore not mandatory for compliance with this specification. However, if these features are implemented, they MUST adhere to the protocol described herein in to be compliant. These features include:</p>
```
Comment (SamWhited) [inline xep-0060.xml]: Should read: "**an** XMPP-based pubsub service"

**Comment by sstrigler** [comment]
D'oh, context! Thanks! And fixed.

**Comment by SamWhited** [comment]
@sstrigler One more little thing that I forgot to ask; can you add a `<revision>` block with the others near the top? This only needs a minor version bump since it's an editorial change, so teh correct revision would be `1.13.1`. Thanks!

**Comment by sstrigler** [comment]
@SamWhited done, added my intitials (ss?)

**Comment by SamWhited** [comment]
Thanks! I'll try to get this out soon.

**Comment by SamWhited** [comment]
Merged; as always, sorry for the delay.

**Comment by sstrigler** [comment]
No worries

**Pull request merged**